packages/api/tests/pathfinding.test.ts

```
@@ -0,0 +1,123 @@
+import { PathfindingService } from '../src/pathfinding';
+import { DatabaseService } from '../src/db/connection';
```

ESLint

[error] 'DatabaseService' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)